keylime/tpm/tpm_main.py

```diff
@@ -144,7 +145,7 @@ def __get_pcrs(self):
             raise Exception("get_tpm_algorithms failed with code " + str(code) + ": " + str(errout))
 
         retyaml = config.yaml_to_dict(output, logger=logger)
-        pcrs = {}
+        pcrs: Mapping[str, str] = collections.ChainMap()
```

Comment:
Have to use Mapping here instead of ChainMap since ChainMap on python 3.6 doesn't seem to allow `[str, str]`.